test/helper_non-windows_test.go

```diff
@@ -0,0 +1,10 @@
+// +build !windows
```

Comment:
file name should be `_unix_test.go` check what pattern we use in `engine/native/system/` package.